src/basic/meson.build

```diff
@@ -106,7 +106,6 @@ basic_sources = files(
         'uid-classification.c',
```

Comment:
commit message of the 1st commit: 'currectly' -> 'currently' ?

Note, sd-xyz.h can be included in src/basic/*.c.